packages/components/docs/token-table.tsx

```diff
@@ -11,12 +11,26 @@ type TokenTableProps = {
   renderExample: (name: string, value: string) => JSX.Element;
 };
 
+const StyledTable = styled.table`
```

Comment:
very open to naming suggestions - not a super fan of this name - I could also do a css block instead.

Reply:
Seems reasonable. Could also be something like `TokensTable`

Reply:
The problem was the name overlap with `TokenTable` so I'd rather not have it be a deceptively similar name.

I'll merge for now, and we can discuss the naming conventions for this sort of thing later.